spec/models/user_spec.rb

```diff
@@ -29,5 +29,15 @@
       user.reload
       expect(user.television_shows).to eq [television_show]
     end
+
+    it "returns falsy if the user already recommended it" do
```

Comment:
Very minor personal preference, but I usually spot an `if` in a description and extract it to a `context` block. Not sure if that means anything other than reminding me to test all logical forks. Your call, I'm like a 2/10 on that.